collections/_patterns/email-address.html

```diff
@@ -0,0 +1,71 @@
+---
+title: Email Address
+description: Use this pattern when you need to ask users for their email address.
+react-link: https://1-0-0-dev.dg2jutx9gylat.amplifyapp.com/?path=/docs/components-form-patterns--email#email-pattern
```

Comment:
could u help to create a constant in _config.yml and store the domain of storybook as https://1-0-0-dev.dg2jutx9gylat.amplifyapp.com . Reference the domain url using string literals together with the path

Update all files using this same domain link